rust/lance-index/src/vector/ivf/builder.rs

```diff
@@ -40,8 +41,15 @@ pub struct IvfBuildParams {
 
     pub sample_rate: usize,
 
+    /// Precomputed partitions file (row_id -> partition_id)
+    /// mutually exclusive with `precomputed_shuffle_buffers`
     pub precomputed_partitons_file: Option<String>,
 
+    /// Precomputed shuffle buffers (row_id -> partition_id, pq_code)
```

Comment:
Should we just deprecate one precompute partition file?

Just leave one of these two.